CHANGELOG.template.md

```diff
@@ -4,11 +4,17 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-### {{Major.Minor.Patch}}
+### [1.0.1](https://github.com/plus3it/actions-workflows/releases/tag/1.0.1)
 
-**Commit Delta**: {{ Link to compare changes with prior version }}
+**Released**: 2023.02.06
 
-**Released**: {{ YYYY.MM.DD }}
+**Summary**:
+
+*   Allows test job to be optional in release workflow 
+
+### [1.0.0](https://github.com/plus3it/actions-workflows/releases/tag/1.0.0)
+
+**Released**: 2023.01.24
```

Comment:
This can all be deleted. This is the changelog _template_, it is not the changelog for this project, that would be `CHANGELOG.md`...

```suggestion
```

Reply:
Removed